notebooks/03_raster_zonal_stats.ipynb

```diff
@@ -354,8 +354,8 @@
      "name": "stdout",
```

Comment:
<p>Should this be <code>create_exactextract_zonal_stats()?</code> I think there are 3 instances of <code>get_exactextract_zonal_stats()</code> in this notebook.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/geowrangler/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>